mkdocs.yml

```diff
@@ -28,6 +28,7 @@ nav:
     - KubeRay Operator: components/operator.md
     - KubeRay ApiServer: components/apiserver.md
     - KubeRay CLI: components/cli.md
+    - Kuberay with MCAD: deploy/kuberay-with-MCAD.md
```

Comment:
Kuberay -> KubeRay

With the current doc structure, the new page fits better 
- Under guidance in the file structure: `guidance/kuberay-with-MCAD.md`
- Under Features in the table of contents

Reply:
Done, thanks!